src/asio_client_session.cc

```diff
@@ -44,6 +44,13 @@ session::session(boost::asio::io_service &io_service, const std::string &host,
   impl_->start_resolve(host, service);
 }
 
+session::session(boost::asio::io_service &io_service, const boost::asio::ip::tcp::endpoint &local_endpoint,
+                 const std::string &host, const std::string &service)
+    : impl_(std::make_shared<session_tcp_impl>(
+          io_service, host, service, boost::posix_time::seconds(60))) {
+  impl_->start_resolve(host, service);
```

Comment:
local_endpoint is still not used.  Should I assume that you just added session_tcp_impl::session_tcp_impl ctor which takes local_endpont and didn't fully implement it so that it can be used by public interface?

Reply:
Tatsuhiro, thanks for your patience. I keep doing the merge wrong.

Best
A.